src/transformers/modeling_tf_utils.py

```diff
@@ -2166,8 +2169,10 @@ def save_pretrained(
             files_timestamps = self._get_files_timestamps(save_directory)
 
         if saved_model:
+            if not signatures:
```

Comment:
```suggestion
            if signatures is not None:
```
(same thing in practice, but might help catching incorrect falsy values in `signatures`)